controllers/ibmblockcsi_controller.go

```diff
@@ -308,6 +317,17 @@ func (r *IBMBlockCSIReconciler) updateStatus(instance *ibmblockcsi.IBMBlockCSI,
 		return err
 	}
 
+	callHomeCronJob, err := r.getCallHomeCronJob(instance)
+	if err != nil && errors.IsNotFound(err) {
+	} else if err != nil {
+		return err
+	} else if !r.isCallHomeDefined(instance) {
+		err = r.deleteCallHome(instance, callHomeCronJob, logger)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
why is this inside `updateStatus`?

Reply:
cause there is where the other object getting restarted by deleting their pod..

Reply:
how is restart related to deletion?

Reply:
you can say its related cause the call home status is turned to disable

Reply:
that is a change in cr, not the actual status of the deployment